docs/coordinates/index.rst

```diff
@@ -340,6 +340,19 @@ To see the list of site names available, use
     >>> EarthLocation.get_site_names()  # doctest: +REMOTE_DATA
     ['ALMA', 'AO', 'ARCA', ...]
 
+Both :func:`~astropy.coordinates.EarthLocation.of_site` and
+:func:`~astropy.coordinates.EarthLocation.get_site_names`,
```

Comment:
```suggestion
Both :meth:`~astropy.coordinates.EarthLocation.of_site` and
:meth:`~astropy.coordinates.EarthLocation.get_site_names`,
```

Reply:
Moved the references to ``:func:`` to ``:meth:`` in this section of the documentation.